.gitignore

```diff
@@ -128,6 +128,7 @@ collectors/python.d.plugin/python.d.plugin
 collectors/fping.plugin/fping.plugin
 collectors/ioping.plugin/ioping.plugin
 collectors/go.d.plugin
+web/netdata-switch-dashboard.sh
 
 # installer generated files
 /netdata-uninstaller.sh
```

Comment:
Play with moving a couple of charts to show the future and hitting the play/reset button a few times. Most of the time play/refresh doesn't work. Eventually, I get the following error 3 times and the dashboard crashes. (Ubuntu 18.04, Firefox 74.0)
```
TypeError: "d is undefined"
    Ne use-show-value-outside.ts:71
    Ne use-show-value-outside.ts:70
    React 12
    Redux 5
    X chart.tsx:129
    highlightCallback dygraph-chart.tsx:420
    mouseMove_ dygraph.js:1621
    mouseMoveHandler_ dygraph.js:829
```

It also happened once when I zoomed out 3 times, but couldn't repeat it.

Reply:
this is also fixed, it was small bug related to the feature i added recently (support for `data-show-value-of-` attribute)

Also, if you scroll to the future, i haven't stopped updating, as it was in old dashboard. That means the mode is still "live", but you can see updates in the middle of the chart. When the updates reach the right side of the chart-container, it starts to auto-scrolling again. So the "play" button does nothing in this case (because the "play" mode is "on" anyway). Is that ok to you? Btw. I think users rarely scroll to the future